tests/e2e/test_rosacli_cluster.go

```diff
@@ -1158,6 +1158,31 @@ var _ = Describe("HCP cluster creation negative testing",
 							"External authentication is only supported in version '4.15.9' or greater, current cluster version is '%s'",
 							foundVersion))
 			})
+
+		It("expose additional allowed principals for HCP negative - [id:74433]",
+			labels.Medium, labels.Runtime.Day1Negative,
+			func() {
+				By("create hcp cluster using --additional-allowed-principals and invalid formatted arn")
+				clusterName := "ocp-74408"
+				replacingFlags := map[string]string{
+					"-c":              clusterName,
+					"--cluster-name":  clusterName,
+					"--domain-prefix": clusterName,
+				}
+
+				By("Create cluster with invalid subnets")
+				rosalCommand.ReplaceFlagValue(replacingFlags)
```

Comment:
What is `rosalCommand`? Is it generated from an existing command line file? In day1Negative job, there won't be such a file existing

Reply:
No, it's generating create cluster command from a random profile, and It's needed for hcp cluster